javascript/javascript1/week1/dogAge.js

```diff
@@ -0,0 +1,18 @@
+//2. Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 2016;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
+
+let dogYear;
+if (shouldShowResultInDogYears) {
+  dogYear = dogYearFuture - dogYearOfBirth;
+} else {
+  dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+}
```

Comment:
1. It's a good practice to always set a value to a variable, even when it's initiated as empty. For example you know that `dogYear` will be a number. So here you could set it as `let dogYear = 0;`
2. But on another end you repeat `dogYearFuture - dogYearOfBirth` two times. Why not set already on line 7 `dogYear` to `dogYearFuture - dogYearOfBirth`? Then you can also remove this `if else` and use `dogYear = dogYear * 7;` right before the `console.log` for the true case in the `if else` starting at line 14.